Deploy/stacks/dynamic/stack-clients/src/main/resources/com/cmclinnovations/stack/clients/citydb/citydb_add_building_height.sql

```diff
@@ -0,0 +1,42 @@
+SET
+    search_path TO public,
+    citydb;
+
+UPDATE
+    "building"
+SET
+    "measured_height" = "h"
```

Comment:
is "measured_height" something custom for us? probably worth documenting, at least myself be looking for this to configure the configuration file for visualisation